Detectors/GRP/workflows/src/create-grp-ecs.cxx

```diff
@@ -96,6 +102,7 @@ int main(int argc, char** argv)
     add_option("help,h", "Print this help message");
     add_option("period,p", bpo::value<std::string>(), "data taking period");
     add_option("run,r", bpo::value<int>(), "run number");
+    add_option("run-type,t", bpo::value<int>(int(GRPECSObject::RunType::NONE)), "run type");
```

Comment:
`-t` is also for `triggering`, maybe you could use a different letter?

Reply:
thanks for spotting, fixed